ui/src/components/DiffTextArea/index.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { string, func, number } from 'prop-types';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import { ReactGhLikeDiff } from 'react-gh-like-diff';
+import 'react-gh-like-diff/lib/diff2html.css';
+
+@withStyles(theme => {
+  const borderColor =
+    theme.palette.type === 'light'
+      ? fade(theme.palette.common.black, 0.23)
+      : fade(theme.palette.common.white, 0.23);
+
+  return {
+    tab: {
+      flexGrow: 1,
+      color: theme.palette.text.primary,
```

Comment:
Add `width: '100%'` to make sure the container doesn't go past the screen and avoid having the chance of having two scrollers.

Reply:
Hi @helfi92 I just updated this PR! Thanks for the advices and review :)